Nightly release stopped working

```
/Users/runner/work/_temp/3a790790-3d16-4323-b0be-0fc5e6fb1d70.sh: line 1: /Users/runner/.cargo/bin/cargo: cannot execute binary file
Error: Process completed with exit code 126.
```

--- a/.github/actions/rust/action.yml
+++ b/.github/actions/rust/action.yml
@@ -55,7 +55,6 @@ runs:
       id: rust-cache
       with:
         path: |
-          ~/.cargo/bin/
           ~/.cargo/registry/index/
           ~/.cargo/registry/cache/
           ~/.cargo/git/db/

